src/twisted/application/runner/_pidfile.py

```diff
@@ -0,0 +1,270 @@
+# -*- test-case-name: twisted.application.runner.test.test_pidfile -*-
+# Copyright (c) Twisted Matrix Laboratories.
+# See LICENSE for details.
+
+"""
+PID file.
+"""
+
+import errno
+from os import getpid, kill
+
+from zope.interface import Interface, implementer
+
+from twisted.logger import Logger
+
+
+
+class IPIDFile(Interface):
+    """
+    PID file.
+
+    Manages a file that remembers a process ID.
```

Comment:
I kinda feel like this line by itself is sufficiently descriptive and `"PID file"` above is unnecessary.